integration-libs/s4om/src/s4om.module.ts

```diff
@@ -0,0 +1,8 @@
+import { NgModule } from '@angular/core';
```

Comment:
I would rename `src` directory to `components` to have similar structure to the other libraries (see below):

![image](https://user-images.githubusercontent.com/97286147/169796474-c783817d-ede8-4181-8073-a4098cc28512.png)

Reply:
thanks Michal, will rename the `src` folder as `components`